runtime/src/main.rs

```diff
@@ -31,11 +32,21 @@ const BANNER: &str = r#"
                |_|
 "#;
 
+fn report_fw_load_success() {
+    // Complete the firmware download mailbox command
+    let mbox = mbox::RegisterBlock::mbox_csr();
+    mbox.status()
+        .write(|w| w.status(|_| MboxStatusE::CmdComplete));
```

Comment:
This has already been done. The ROM completed the mailbox transaction write after it validated the signature. Remove this.

Reply:
Done.